### PR TITLE
fix, refactor(egen):  fixes data import issue, includes pipeline job deletion, corrections from template.

### PR DESCRIPTION
**Pull request by SumanthKasula99**
This PR consists of the following updates:

1. Fixes VertexAI image dataset creation by updating the gcs_source link.
2. Includes pipeline job deletion step in cleanup section.
3. Refactors the notebook according to the notebook template (removes boilerplate elements, renames REGION with LOCATION and re-structures the sections).
4. Removes the usage of future tense(will) according to the guidelines.

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/pipelines/google_cloud_pipeline_components_automl_images.ipynb [hunk 1]
```diff
@@ -74,26 +78,26 @@
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you learn how to use `Vertex AI Pipelines` and `Google Cloud Pipeline Components` to build an `AutoML` image classification model.\n",
+        "In this tutorial, you learn how to use Vertex AI pipelines and Google Cloud pipeline components to build an AutoML image classification model.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/google_cloud_pipeline_components_automl_images.ipynb:81]: The product/service is "Vertex AI Pipelines" (note upper case)

File: notebooks/official/pipelines/google_cloud_pipeline_components_automl_images.ipynb [hunk 2]
```diff
@@ -74,26 +78,26 @@
...
         "\n",
         "\n",
         "This tutorial uses the following Google Cloud ML services:\n",
         "\n",
-        "- `Vertex AI Pipelines`\n",
-        "- `Google Cloud Pipeline Components`\n",
-        "- `Vertex AutoML`\n",
-        "- `Vertex AI Model` resource\n",
-        "- `Vertex AI Endpoint` resource\n",
+        "- Vertex AI pipelines\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/google_cloud_pipeline_components_automl_images.ipynb:86]: change to:

- Vertex AI Pipelines

File: notebooks/official/pipelines/google_cloud_pipeline_components_automl_images.ipynb [hunk 3]
```diff
@@ -74,26 +78,26 @@
...
-        "    - Train an AutoML image classification `Model` resource.\n",
-        "    - Create an `Endpoint` resource.\n",
-        "    - Deploys the `Model` resource to the `Endpoint` resource.\n",
+        "    - Create a dataset resource.\n",
+        "    - Train an AutoML image classification model resource.\n",
+        "    - Create an endpoint resource.\n",
+        "    - Deploys the model resource to the endpoint resource.\n",
         "- Compile the KFP pipeline.\n",
-        "- Execute the KFP pipeline using `Vertex AI Pipelines`\n",
+        "- Execute the KFP pipeline using Vertex AI pipelines.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/google_cloud_pipeline_components_automl_images.ipynb:100]: change to:

- Vertex AI Pipelines

File: notebooks/official/pipelines/google_cloud_pipeline_components_automl_images.ipynb [hunk 4]
```diff
@@ -653,11 +605,11 @@
       "source": [
         "Click on the generated link to see your run in the Cloud Console.\n",
         "\n",
-        "<!-- It should look something like this as it is running:\n",
+        "<!-- It should look something like this as it's running:\n",
         "\n",
         "<a href=\"https://storage.googleapis.com/amy-jo/images/mp/automl_tabular_classif.png\" target=\"_blank\"><img src=\"https://storage.googleapis.com/amy-jo/images/mp/automl_tabular_classif.png\" width=\"40%\"/></a> -->\n",
         "\n",
-        "In the UI, many of the pipeline DAG nodes will expand or collapse when you click on them. Here is a partially-expanded view of the DAG (click image to see larger version).\n",
+        "In the UI, many of the pipeline DAG nodes expand or collapse when you click on them. Here is a partially-expanded view of the DAG (click image to see larger version).\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/google_cloud_pipeline_components_automl_images.ipynb:612]: - Is "UI" referencing the "Google Cloud console"? If so, change "UI" to"Google Cloud console"

- change "Here is" to "Here's"

**Review by kittyabs — COMMENTED** [review]
suggested edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**